Tools/Schema.NET.Tool/Models/SchemaClass.cs

```diff
@@ -9,8 +9,8 @@ public class SchemaClass : SchemaObject
     {
         private static readonly Uri EnumerationId = new("https://schema.org/Enumeration");
 
-        public SchemaClass(Uri id, string label, string layer)
-            : base(id, label, layer)
+        public SchemaClass(string comment, Uri id, string label, string layer)
```

Comment:
Minor comment. It would be nice if the order of parameters in these constructos followed the hierarchy of data.

`string layer, Uri id, string label, string comment`

Reply:
Happy to do that. While I know what hierarchy means, I'm not sure exactly how you're deriving it - like these properties exist in the same hierarchy of inheritance. Just want to understand how you want it.

I went alphabetical for this but felt weird to have comment first.

Reply:
I think layer comes first because it groups many types together. The ID for a type makes sense next as it is the unique identifier. After that the label seems to make sense to me, although I can't explain why. After these three, I think going with alphabetical ordering is fine, unless it makes sense to group parameters together somehow.